README.md

```diff
@@ -50,24 +50,24 @@ npm run server
 
 ### Documentation:
 
-Check [docs.mrjs.io](https://docs.mrjs.io) for the full documentation or our [repository](https://github.com/Volumetrics-io/documentation).
+Check [docs.mrjs.io](https://docs.mrjs.io) for the full documentation, or our [repository](https://github.com/Volumetrics-io/documentation).
```

Comment:
We should actually just change this to :

Check [docs.mrjs.io](https://docs.mrjs.io) or our [repository](https://github.com/Volumetrics-io/documentation) for the full documentation.